src/Akeneo/Channel/back/Infrastructure/Symfony/Resources/config/queries.yml

```
@@ -37,3 +37,8 @@ services:
         class: Akeneo\Channel\Infrastructure\Query\DummyIsLocaleEditable
         arguments:
             - '@database_connection'
+
+    Akeneo\Channel\Infrastructure\Query\Sql\SqlGetChannels:
+        class: Akeneo\Channel\Infrastructure\Query\Sql\SqlGetChannels
+        arguments:
+            - '@database_connection'
```

In order to registrate your query into the clearable query 
```suggestion
            - '@database_connection'
        tags: ['akeneo.pim.cached_query']
```